### PR TITLE
Refactored SIGINT handler so it catches SIGINT during relog loop

### PR DESCRIPTION
**Pull request by Quantra**
Refactored SIGINT handler such that it is caught when in [server_error] relogging loop

## Fixes (provide links to github issues if you can):
-
-
-

### CODE REVIEW

**Comment by TheSavior** [comment]
@douglascamata please review

**Comment by Quantra** [comment]
I've tested this with regular SIGTERM restarts for > 8 hours now.

It is catching SIGTERM more reliably than my initial implementation.

File: pokecli.py [hunk 1]
```diff
@@ -139,6 +139,15 @@ def handle_sigint(*args):
 
     except GeocoderQuotaExceeded:
         raise Exception("Google Maps API key over requests limit.")
+    except SIGINTRecieved:
+        if bot:
+            bot.event_manager.emit(
+                'bot_interrupted',
+                sender=bot,
+                level='info',
+                formatted='Bot got SIGTERM. Shutting down.'
```
Comment (TheSavior) [inline pokecli.py]: This should say SIGINT, right? Since that is what you are handling/

**Comment by TheSavior** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/PokemonGoF/PokemonGo-Bot/pull-request/3867/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=PokemonGoF/PokemonGo-Bot)

**Pull request merged**